packages/authgear-web/src/storage.ts

```diff
@@ -52,6 +53,19 @@ export class PersistentTokenStorage implements TokenStorage {
   async delRefreshToken(namespace: string): Promise<void> {
     await this.storageDriver.del(this.keyMaker.keyRefreshToken(namespace));
   }
+}
+
+/**
+ * @internal
+ */
+export class PersistentSharedStorage implements SharedStorage {
```

Comment:
On a second thought, SharedStorage should really be named InterAppSharedStorage. And PersistentSharedStorage should be named SingleAppPersistentInterAppSharedStorage.